h/views/api/bulk.py

```diff
@@ -72,11 +71,6 @@ def _check_authority(self, data_type, body):
             self._assert_authority("groupid", body.attributes["groupid"])
             self._assert_authority("query groupid", body.meta["query"]["groupid"])
 
-        elif data_type == DataType.GROUP_MEMBERSHIP:
-            self._assert_authority(
-                "member id", body.relationships["member"]["data"]["id"]
-            )
-
```

Comment:
Nothing to check for in the group membership any more, they are just integers

Reply:
:+1: